### PR TITLE
Update to match new mmath with functions working only on realised data

### PR DESCRIPTION
**Pull request by starstuck**

### CODE REVIEW

File: src/dqe_trans2.erl [hunk 1]
```diff
@@ -20,7 +20,7 @@ start({_Start, _Count}, State) ->
     {ok, State}.
 
 emit(_Child, {realized, {Data, Resolution}}, State = #state{op = Op, arg = Arg}) ->
```
Comment (elvisci) [inline src/dqe_trans2.erl]: Line 22 is too long: emit(_Child, {realized, {Data, Resolution}}, State = #state{op = Op, arg = Arg}) ->.

**Comment by Licenser** [comment]
Hi mate,
I'm currently working on a complete update on this that encoperates the function table I'll hold of on merging this since it once the function table is ready literally everything here will change.

**Comment by starstuck** [comment]
OK, I had also some change in mind, to get rid of RDATA_SIZE macro include, but I can hold off until you are done. Meanwhile I'll fix elvis objections.

**Comment by Licenser** [comment]
I've pusehd the current broken, undocumented and incimplete (read going tio ship tomorrow [just kidding]) state of things to the funtab brunch.

**Comment by starstuck** [comment]
I'll have a look. I guess we can discuss detail tomorrow afternoon during your visit.

**Comment by Licenser** [comment]
I was hoping to :)

**Pull request merged**